Update README.md (#1)
ccmmf/.github: profile/README.md

# California Cropland Monitoring and Modeling Framework

## About

> The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 

Reviewer: "to generate historical inventories and future projections"?

Author: @serbinsh its not clear - what is the question? Does the wording seem odd or confusing? 

Goal is to convey that we will generate:
1. Annual inventories of SOC and GHG fluxes and
2. Future projections of SOC and GHG fluxes under different agronomic management and climate scenarios

Reviewer: my idea here was to be more specific since just saying inventories doesnt exactly clarify the timeline or if this means from now on? for a historical period?  so historical was meant to be added to clarify

Author: well, it is only 'historical' in the sense that each year we are creating an inventory ('reanalysis???') for the year that has just ended.

Author: ```suggestion
The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate current year inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 
```

> CCMMF will leverage Bayesian techniques to combine heterogeneous datasets into unified wall-to-wall inventories and climate and scenario-based projections. 

Reviewer: "to combine model predictions and heterogenous datasets..." ?

Author: ```suggestion
CCMMF will leverage Bayesian techniques to combine model predictions and heterogeneous datasets into unified wall-to-wall inventories and climate and scenario-based projections. 
```

> Data layers will be generated as ensemble projections to facilitate propagation of uncertainty through downstream applications.

Reviewer: "The data layers generated in our model-data integration framework..." "..though downstream applications, including those designed to inform land management and climate policy"?

> CCMMF will produce new gridded time series of key agronomic practices including planting, harvest, irrigation, and tillage derived from satellite 

Author: ```suggestion
CCMMF will produce new gridded time series of key agronomic practices including planting, harvest, irrigation, and tillage at a consistent temporal resolution. 
These agronomic management time series will be generated by combining remotely sensed data with agronomic statistics and biogeochemical modeling.
```

> measurements and agronomic statistics. 

Reviewer: "CCMF will produce new gridded time-series information on ..... at a consistent spatial and temporal resolution of XX and monthly periods, respectively"? 

And this doesnt seem to capture the role of the model as well...perhaps "by combining remotely sensed data, agrinomic statistics, and biogeochemical modeling" or something like that?

Author: ```suggestion
```

> The biogeochemistry model will be built on SIPNET, expanded to represent agronomic management, nitrogen cycling, and fluxes of nitrous oxide and methane. 

Reviewer: "...SIPNET (the simplified photosynthesis and evapotranspiration model)..."

Author: ```suggestion
The biogeochemistry model will be built on the SImplified PhotosyNthesis and EvapoTranspiration model (SIPNET), expanded to represent agronomic management, nitrogen cycling, and fluxes of nitrous oxide and methane. 
```

Reviewer: Looks good

> Using remotely sensed practices to drive the model will produce consistent model estimates that do not depend on obtaining records from individual farmers. 

Reviewer: "We will utilize existing and newly-derived remote sensing of annual land management and agricultural practices ..."

Author: ```suggestion
We will utilize existing and newly-derived remote sensing of annual land management and agricultural practices to drive the model will produce consistent model estimates that do not depend on obtaining records from individual farmers. 
```

Reviewer: Looks good

The statistical workflow engine will extend the Predictive Ecosystem Analyzer (PEcAn) to support annual updates and both climate-based and management scenario-based projections.

The combination of a simplified biogeochemistry model that simulates agronomic practices from remote-sensed inputs and an open, consistent, variance-explicit data framework will allow CCMMF to achieve robust estimates of SOC and GHG inventories across lands with highly varied but coarsely measured management. 
All data and software that is part of CCMMF will be open, free, and deployable on state computing resources. 
This requirement sets a bar for transparency, and a foundation for future innovation and transferability, that is not possible with the current suite of proprietary systems.

> ## People 

Reviewer: Should we add our affiliations here?

Author: done in e9a4c9b958d6ddf4eb62060b7bea6e27a922870a

Chris Black
David LeBauer
Mike Dietze
Rob Kooper
Mike Longfritz
Shawn Serbin

## Contributing

> ## Useful Resources 

Reviewer: Do we also want a "Research Partners and External Stakeholders" list here?  Maybe list NASA and some others herer?

Do we have a funding support part in this as well? Who is funding under what contract?  

Should we include the unifying workflow figure in the readme as well that shows graphically how it alll fits together?

Author: all good suggestions!

Author: added to #4
